packages/gatsby-theme-newrelic/src/components/Callout.js

```diff
@@ -11,16 +11,28 @@ const VARIANTS = {
 
 const styles = {
   variant: {
+    [`${VARIANTS.CAUTION}-title`]: css`
+      color: var(--color-red-400);
+    `,
     [VARIANTS.CAUTION]: css`
-      border-left: 4px solid var(--color-red-400);
+      border: 1px solid var(--color-red-400);
+      border-left-width: 6px;
       background: var(--callout-caution-background-color);
     `,
+    [`${VARIANTS.IMPORTANT}-title`]: css`
+      color: var(--color-yellow-400);
+    `,
     [VARIANTS.IMPORTANT]: css`
-      border-left: 4px solid var(--color-yellow-400);
+      border: 1px solid var(--color-yellow-400);
+      border-left-width: 6px;
       background: var(--callout-important-background-color);
     `,
+    [`${VARIANTS.TIP}-title`]: css`
+      color: var(--color-green-400);
+    `,
     [VARIANTS.TIP]: css`
-      border-left: 4px solid var(--color-green-400);
+      border: 1px solid var(--color-green-400);
+      border-left-width: 6px;
```

Comment:
Now that I see this, there is one more bit of conslidation we could probably do 😄 . Since we know all variants will have the same border, just with different border colors, we could extract the common bit to the main style below and have this only change the border color.

```
[VARIANTS.CAUTION]: css`
  // ...
  border-color: var(--color-red-400);
`,
[VARIANTS.TIP]: css`
  // ...
  border-color: var(--color-green-400);
`
```

Further down:

```
css={css`
  // ...
  border: 1px solid;
  border-left-width: 6px;
  ${styles.variant[variant]}
`}
```

Reply:
oh yes indeed. good call.